src/gui_common/GUICommon.cs

```diff
@@ -1,5 +1,7 @@
-﻿using Godot;
-using Godot.Collections;
+﻿using System;
+using Godot;
+using Array = Godot.Collections.Array;
+using Object = Godot.Object;
```

Comment:
With the exception throwing gone, I think these changes can be reset.

Reply:
Looks like it wants two aliases over a single System.Math call: https://app.circleci.com/pipelines/github/Revolutionary-Games/Thrive/4105/workflows/209b4038-25c7-4d43-9ecf-b0fa9b6ed564/jobs/13356

This is your code on linters.

Reply:
I like the fact that in actual code body you aren't allowed to use the full name to refer to things, I think that unnecessarily makes the code harder to read.